index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <nav>
+      <ul>
+        <li>Home<a href=""></a></li>
+        <li>Speakers<a href=""></a></li>
+        <li>Events<a href=""></a></li>
+        <li>Details<a href=""></a></li>
+      </ul>
+    </nav>
+    <h1>Artist should get free support animals</h1>
+    <img
+            src="https://cdn.glitch.com/0f06e0d6-dc97-4290-b900-077fbd9b464c%2FMike-Andrews-300x300.jpg?v=1607221475535"
+          />
+    <p>
+      Mr. Andrews is Managing Partner of Andrews & Associates, a consulting firm
+      specializing in new product R&D, business and management development,
+      Emergency Planning, Business Mitigation, Response and Resumption Planning
+      and Operational Readiness programs for business, schools and government
+      entities.
+    </p>
+    <img
+            src="https://cdn.glitch.com/0f06e0d6-dc97-4290-b900-077fbd9b464c%2Fcatherine-jenkins-300x300.jpg?v=1607221473329"
+          />
+    <p>
+      Dr. Kate Jenkins is a software engineer for Ejenta, Inc, a startup in San
+      Francisco that uses artificial intelligence to monitor and reason about
+      behavioral information (human-human and human-computer). For projects
+      spanning industry verticals from space to government to healthcare, she is
+      responsible for hardware integration and sensor streaming (proximity,
+      gesture, social dynamics). She co-founded her first company in 2004 and
+      has since published more than 35 peer-reviewed articles on advanced
+      materials for magnetoelectric sensing and high performance computing. She
+      is a Senior Member of IEEE (magnetics, CES) and in 2015-2016 will be the
+      student activities chair for IEEE Region 6
+    </p>
+    <img
+            src="https://cdn.glitch.com/0f06e0d6-dc97-4290-b900-077fbd9b464c%2Fcat-tie.jpg?v=1607221338834"
+          />
+    <p>
+      Mr. Panther is assistant head for the Cat Nap Association or CNA for
+      short. CNA is based in New Jersey. He is also one of the founder members.
+      CNA was founded in 2010.
+    </p>
+    <p><button type="#top">home</button></p>
```

Comment:
This should be a link and not a button element; also, there are only three valid `type` options for a `button`: reset, submit and button 👍🏻

```suggestion
    <p><a href="#top">home</a></p>
```